src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,32 @@
+package seedu.addressbook.data.tag;
+
+public class Tagging {
+	private char sign;
+	private String personName;
+	private String tag;
+	
+	public Tagging() {
+	}
+	
+	public Tagging(char sign, String personName, String tag) {
```

Comment:
I can't agree that this is an Association class because you are not maintaining the Person and the Tag classes, which the Tagging class keeping information about. 
Hint: Consider a constructor as Tagging(boolean sign, Person person, Tag tag)
Have private variables to maintain these instances.
Have getters for the private variables.

Char is not a good type to maintain the tagging operation. Consider a boolean or an enum